test_scripts/ATF_ShutDown_IGNITION_OFF.lua

```diff
@@ -0,0 +1,71 @@
+-- Requirement summary:
+-- [Data Resumption]:OnExitAllApplications(IGNITION_OFF) in terms of resumption 
+--
+-- Description:
+-- 1. HMI is running
+--    One App is registered and activated on HMI
+-- 2. Perform ignition Off
+--    HMI sends OnExitAllApplications(IGNITION_OFF)
+--
+-- Expected result:
+-- 1. SDL sends to App OnAppInterfaceUnregistered
+-- 2. SDL sends to HMI OnSDLClose and stops working
+
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+function Test:Start_SDL_With_One_Activated_App()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
```

Comment:
@LitvinenkoIra , i don't know why we have to put many actions  and their expected result into a Test. If one Test only contains one action and expected result, our script will be easy to read and understand

Current code:
```
function Test:Start_SDL_With_One_Activated_App()
  self:runSDL()
  commonFunctions:waitForSDLStart(self):Do(function()
    self:initHMI():Do(function()
      commonFunctions:userPrint(35, "HMI initialized")
      self:initHMI_onReady():Do(function ()
        commonFunctions:userPrint(35, "HMI is ready")
        self:connectMobile():Do(function ()       
          commonFunctions:userPrint(35, "Mobile Connected")
          self:startSession():Do(function ()
            commonSteps:ActivateAppInSpecificLevel(self, 
              self.applications[config.application1.registerAppInterfaceParams.appName])
            EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
          end)
        end)
      end)
    end)
  end)
end
```
Suggestion: Each Test contains only one action and expected result.
```
function Test:Start_SDL()
  self:runSDL()
  commonFunctions:waitForSDLStart(self)
end
function Test:InitHMI()
  self:initHMI()
end
function Test:InitHMI()
  self:initHMI_onReady()
end
function Test:ConnectMobile()
  self:connectMobile()
end
function Test:StartSession()
  self:startSession()
end
function Test:StartSession()
  commonSteps:ActivateAppInSpecificLevel(self, 
    self.applications[config.application1.registerAppInterfaceParams.appName])
  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
end

```